### ISSUE TITLE
[Descriptor Issue]: antsApplyTransforms

### ISSUE THREAD

**Issue by kaitj** [childmindresearch/niwrap#160]
### Package

ANTs

### Tool Name

ants.apply_transforms

### Problem Description

The compiled version of antsApplyTransforms has logic that is causing the command line to return nothing (e.g. `cargs = []`) for the interpolation method (e.g. `--interpolation `). as it expects both optional arguments to be present.

### Steps to Reproduce

Call `antsApplyTransforms` interface with any interpolation method.

Affected methods:
- MultiLabel
- Gaussian
- BSpline
- GenericLabel

Example from compiled code:

```python
cargs = []
        if self.sigma is not None or self.alpha is not None:
            cargs.append("MultiLabel[sigma=" + (str(self.sigma) if (self.sigma is not None) else "") + ",alpha=" + (str(self.alpha) if (self.alpha is not None) else "") + "]")
        return cargs
```

Should pass `MultiLabel` without brackets if no options provided.

### Logs

_(Removed incorrect log, will update with correct one. [Nov 14, 2024])_

**Comment by kaitj** [comment]
This seems to have been fixed with `0.3.x` - will reopen if error is encountered again.

**Comment by kaitj** [comment]
This is fixed (can't find the commit that fixed this, but will add if I come across it again). Note, that for those interpolation methods that are affected, need to use the `*noparam` version (e.g. for `MultiLabel` interpolation -> `ants.AntsApplyTransformsMultiLabelnoparams()`)
